tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_friendship.c

```diff
@@ -815,6 +815,26 @@ static void test_other_group(void)
 	PASS();
 }
 
+/** LPN terminate cb test.
+ *
+ * Check that terminate cb is not triggered when there is no established
+ * connection.
+ */
+static void test_lpn_term_cb_check(void)
+{
+	bt_mesh_test_setup();
+
+	bt_mesh_lpn_set(true);
+	k_sleep(K_MSEC(400));
```

Comment:
Could this be replaced with `evt_wait(LPN_POLLED, K_MSEC(...))`? Timeout could be some reasonable value, for example, number of Friend Requests including delay for response.